src/main/kotlin/racingcar/infrastructure/ConsoleOutput.kt

```diff
@@ -0,0 +1,31 @@
+package racingcar.infrastructure
+
+import racingcar.domain.Car
+
+class ConsoleOutput : Output {
+  override fun showMessage() {
+    println(PLAY_RESULT)
+  }
+
+  override fun showRoundResult(cars: List<Car>) {
+    var result = ""
+    for (car in cars) {
+      val carName: String = car.name
+      val carDistance: String = ONE_STEP.repeat(car.distance)
+      result += "$carName : $carDistance\n"
+    }
+    println(result)
+  }
+
+  override fun showWinners(winners: List<String>) {
+    val result = winners.joinToString(DELIMITER)
```

Comment:
현재 상수 DELIMITER에 `,`가 들어있는데 출력 예시에는 `, `와 같이 콤마+공백으로 연결되어 있었습니다
```suggestion
    val result = winners.joinToString()
```
`, `를 상수로 지정하거나 파라미터를 생략하면 자동으로 `, `로 연결됩니다!

Reply:
", 를 상수로 지정하거나 파라미터를 생략하면 자동으로 , 로 연결됩니다!" -> 제가 놀랐던 부분이네요! 감사합니다!